DependencyInjection/EkinoNewRelicExtension.php

```diff
@@ -71,6 +72,20 @@ public function load(array $configs, ContainerBuilder $container)
             $container->removeDefinition('ekino.new_relic.command_listener');
         }
 
+        if ($config['enabled'] && $config['log_logs']['enabled']) {
+            if (!class_exists(NewRelicHandler::class)) {
+                throw new \LogicException('The "monolog/monolog" package must be installed in order to use "log_logs" option.');
+            }
+            $container->getDefinition($config['log_logs']['service'])
+                ->replaceArgument(0, $this->levelToMonologConst($config['log_logs']['level']))
+                ->replaceArgument(2, $config['application_name']);
+
+            foreach ($config['log_logs']['channels'] as $channel) {
+                $def = $container->getDefinition($channel === 'app' ? 'monolog.logger' : 'monolog.logger.'.$channel);
```

Comment:
Could you really modify other bundles services like this?

Reply:
no yhou're right, I can only if the bundle is initialized in the right order. I've to write a compiler pass